v2/samples/dataprotection/v1api/v1api20230101_backupvaultsbackupinstance.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: dataprotection.azure.com/v1api20230101
```

Comment:
The folder this file is in should be `v1api20230101`, not `v1api`, plus you're still missing the recording of the sample being verified.

See [_Run test for added sample and commit the recording_](https://azure.github.io/azure-service-operator/contributing/add-a-new-code-generated-resource/#run-test-for-added-sample-and-commit-the-recording) in our contributor docs.

Reply:
Excluding sample recording for [backupvaultsbackupinstance.yaml] as role assignment creation is not working as expected in samples. Discussed and closed with harsh.